scripts/backend_tests.py

```diff
@@ -55,6 +55,11 @@
     help='optional subdirectory path containing the test(s) to run',
     type=str)
 _PARSER.add_argument(
+    '--exclude_load_tests',
+    help='optional subdirectory which contain the load tests for various' +
```

Comment:
There should be a space after "various". And, "which contain" --> "containing".

But, this docstring is wrong, actually? The arg should be a bool? So the help docstring should be something like "optional; if specified, exclude load tests from being run"?